### DISCUSSION TITLE
Height of a row in gridview

### DISCUSSION THREAD

**Discussion by alex1976** [serenity-is/Serenity#5656] [Help] [answered]
Hi,
How can i set Height of a row in gridview (in EntityGrid.ts or EntityColumns.cs or other location) ?

Thanks all for any answer !

**Comment by minhhungit** [comment, marked as answer]
Check this example https://github.com/volkanceylan/Serene/blob/master/Serene/Serene.Core/Modules/BasicSamples/Grids/InlineImageInGrid/InlineImageInGrid.ts

**Reply by alex1976** [reply]
This is what i need

Thank you very much!
